jenkins-pipelines/oss/longevity/longevity-aws-custom-d2-workload1-multidc-big.jenkinsfile

```diff
@@ -0,0 +1,12 @@
+#!groovy
+
+// trick from https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/43
+def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
+
+// NOTE: multiple real regions must be in alphabetical order for the 'customd_d2/workflow1' case
+longevityPipeline(
+    backend: 'aws',
+    region: '''["eu-central-1", "eu-north-1", "eu-west-1", "eu-west-2", "us-east-1"]''',
+    test_name: 'longevity_test.LongevityTest.test_custom_time',
+    test_config: 'test-cases/longevity/longevity-aws-custom-d2-workload1-multidc-big.yaml',
```

Comment:
the `big` here is a bit ambiguous (number of nodes ?, number of regions ?, big instances  ?)

maybe `full-size` ? 
or `5dcs` vs. `3dcs` ?

i.e. it would be a nickname that we'll be able to reference, ( it won't hold all of the details )

Reply:
Ok, will rename to 5dcs and 3dcs

Reply:
Done